explorer/client/src/pages/transaction-result/TransactionView.tsx

```diff
@@ -150,9 +158,24 @@ type TxItemView = {
 function ItemView({ data }: { data: TxItemView }) {
     return (
         <div className={styles.itemView}>
-            <div className={styles.itemviewtitle}>{data.title}</div>
+            <div
+                className={
+                    data.titleStyle
+                        ? styles[data.titleStyle]
```

Comment:
I don't love this dynamic approach to styling given the definition of the `titleStyle` is not in this file. Given it's only used for on thing, could we maybe just move this to a boolean prop that controlled the application of `styles.itemviewtitle` and `styles.itemfieldstitle`?

Reply:
None of the definitions for styles are ever in the same file as we have it set up